README.md

```diff
@@ -40,8 +40,28 @@ This is a portable library so you can use it anywhere.
 This library should be available on NuGet for easy consumption by your projects.
 You may also copy and paste the P/Invoke signatures you need directly into your projects if desired. 
 
+Packages:
+
+Library      | Package name     | NuGet       | Description
+-------------|------------------|-------------|-------------
+kernel32.dll |`PInvoke.Kernel32`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.Kernel32.svg)](https://www.nuget.org/packages/PInvoke.Kernel32)|Windows Kernel API
+user32.dll   |`PInvoke.User32`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.User32.svg)](https://www.nuget.org/packages/PInvoke.User32)|Windows User Interface
+gdi32.dll    |`PInvoke.Gdi32`   | [![NuGet](https://img.shields.io/nuget/v/PInvoke.Gdi32.svg)](https://www.nuget.org/packages/PInvoke.Gdi32)|[Windows Graphics Device Interface][Gdi]
+setupapi.dll |`PInvoke.SetupApi`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.SetupApi.svg)](https://www.nuget.org/packages/PInvoke.SetupApi)|[Windows application programming interface][SetupApi]
+hid.dll      |`PInvoke.Hid`     | [![NuGet](https://img.shields.io/nuget/v/PInvoke.Hid.svg)](https://www.nuget.org/packages/PInvoke.Hid)|[Windows Human Interface Devices][Hid]
+bcrypt.dll   |`PInvoke.BCrypt`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.BCrypt.svg)](https://www.nuget.org/packages/PInvoke.BCrypt)|[Windows CNG Cryptographic Configuration Functions][BCrypt]
+ncrypt.dll   |`PInvoke.NCrypt`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.NCrypt.svg)](https://www.nuget.org/packages/PInvoke.NCrypt)|[Windows CNG Cryptographic Primitive Functions][NCrypt]
+
+
+
 ## Contribution
 
 Please consider [contributing](CONTRIB.md) more P/Invoke method signatures to this project.
 Once you contribute, you can immediately consume your additions without waiting for another
 public release of the library.
+
+[BCrypt]: https://msdn.microsoft.com/en-us/library/windows/desktop/bb204774
+[NCrypt]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa833130
```

Comment:
The NCrypt link should be https://msdn.microsoft.com/en-us/library/windows/desktop/aa376208(v=vs.85).aspx
The one you've got is actually another BCrypt link.